doc/install_ninja_rst.inc

```diff
@@ -0,0 +1,19 @@
+It is highly recommended to use `Ninja` as the make system under
+Windows. Not only has `Ninja` observed to be faster than MSVC `nmake`, it also
```

Comment:
The phrase `Not only has Ninja observed to be faster ...` doesn't make sense.  Perhaps `Not only has Ninja been observed to be faster ...` is what is meant.

Reply:
It took me 3 reads to see the missing 'been'. I have now added it! Must be one of those optical illusions.